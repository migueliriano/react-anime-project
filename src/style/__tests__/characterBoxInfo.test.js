import React from 'react';

import {
  CharacterBox,
  CharacterName,
  CharacterImage,
} from '../characterBoxInfo';

describe('characterBoxInfo style components', () => {
  it('Should render a CharacterBox Component', () => {
    const tree = renderer.create(<CharacterBox />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a CharacterName Component', () => {
    const tree = renderer.create(<CharacterName />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('Should render a CharacterImage Component', () => {
    const tree = renderer.create(<CharacterImage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
