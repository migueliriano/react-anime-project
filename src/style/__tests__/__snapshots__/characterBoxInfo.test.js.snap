// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`characterBoxInfo style components Should render a CharacterBox Component 1`] = `
.c0 {
  text-align: center;
  margin-right: 20px;
  margin-bottom: 20px;
}

<div
  className="c0"
/>
`;

exports[`characterBoxInfo style components Should render a CharacterImage Component 1`] = `
.c0 {
  max-width: 100px;
  height: 150px;
}

<img
  className="c0"
/>
`;

exports[`characterBoxInfo style components Should render a CharacterName Component 1`] = `
.c0 {
  font-weight: bold;
  max-width: 100px;
  line-height: 20px;
}

<div
  className="c0"
/>
`;
